map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -247,6 +267,26 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
         }
     }
 
+    @UiThread
+    protected fun deriveBaseOverlayDescriptor(host: Activity): BitmapDescriptor {
```

Comment:
This should go in a helper method somewhere for the map image. maybe an `ImageExtensions` class?.
Also use lowest denominator, in this case swap `Activity` with `Context`.

Reply:
(for anyone watching, @agrosner confirmed that offline)

Alright, the new commits have that change. Hopefully it's all good.